src/data/creator.ts

```diff
@@ -2,21 +2,22 @@ import { isValidStringProperty } from '../validation/string';
 import Connection from '../connection';
 import { ObjectsPath } from './path';
 import { CommandBase } from '../validation/commandBase';
+import { Properties, WeaviateObject } from '../openapi/types';
 
 export default class Creator extends CommandBase {
   private className?: string;
   private consistencyLevel?: string;
   private id?: string;
   private objectsPath: ObjectsPath;
   private properties?: any;
```

Comment:
Here do we have some sort of a type for `Properties`?

Reply:
Yes, fixed